x-pack/plugins/upgrade_assistant/public/application/app.tsx

```diff
@@ -52,94 +48,71 @@ const AppHandlingClusterUpgradeState: React.FunctionComponent = () => {
 
   if (missingManageSpacesPrivilege) {
     return (
-      <EuiPageContent
-        verticalPosition="center"
-        horizontalPosition="center"
-        color="subdued"
-        data-test-subj="missingKibanaPrivilegesMessage"
-      >
-        <NotAuthorizedSection
-          title={
-            <FormattedMessage
-              id="xpack.upgradeAssistant.app.deniedPrivilegeTitle"
-              defaultMessage="Kibana admin role required"
-            />
-          }
-          message={
-            <FormattedMessage
-              id="xpack.upgradeAssistant.app.deniedPrivilegeDescription"
-              defaultMessage="To use Upgrade Assistant and resolve deprecation issues, you must have access to manage all Kibana spaces."
-            />
-          }
-        />
-      </EuiPageContent>
+      <NotAuthorizedSection
```

Comment:
Note: This is fine for now. However, it will not render as expected until the `NotAuthorizedSection` component in `es_ui_shared` is updated to use `EuiPageTemplate.EmptyPrompt`. [Related code](https://github.com/elastic/kibana/blob/main/src/plugins/es_ui_shared/__packages_do_not_import__/authorization/components/not_authorized_section.tsx#L18).

This should be addressed via https://github.com/elastic/kibana/issues/163018